taskcluster/configs/config.ci.yml

```diff
@@ -10,11 +10,11 @@ experiment:
   teacher-ensemble: 1
 
   mono-max-sentences-src:
-    total: 10000
```

Comment:
FYI, This is a behavior change for CI, but I did it to match the truncation for the corpus size.